src/plugins/condition/components/ConditionCollection.vue

```diff
@@ -149,8 +154,14 @@ export default {
         observeForChanges() {
             this.stopObservingForChanges = this.openmct.objects.observe(this.domainObject, 'configuration.conditionCollection', (newConditionCollection) => {
                 this.conditionCollection = newConditionCollection;
+                this.updateDefaultCondition();
             });
         },
+        updateDefaultCondition() {
+            const defaultCondition = this.domainObject.configuration.conditionCollection
+                .filter(conditionConfiguration => conditionConfiguration.isDefault)[0];
```

Comment:
`.find()` could be used here, which will avoid the need to fetch the zeroeth element, and is also O(log n) in the typical case, rather than O(n). It's a meaningless optimization in this case, but it's also a tiny change, so maybe we should just do it?

Reply:
True. Would avoid creating a pointless array. It actually wouldn't be optimized though...

The default condition should always be the last element. You cannot reorder it. Should we just use that? Or, find would be a safer lookup in case anything ever changes.

Reply:
@davetsay Oh! Well sure, if it's always the `.length-1`-th element then just directly index it and skip the search.

Reply:
Nevermind. Using find. I haven't been able to successfully reproduce it but I have seen the default condition become out of order. So just in case...